cmd/file_convert.go

```diff
@@ -114,8 +114,14 @@ can be converted into a 'kong-gateway-3.x' configuration file.`,
 		Args: validateNoArgs,
```

Comment:
The new valid formats should be added to suggestions when `deck file convert --help` is run?